crates/nu-std/tests/test_iter.nu

```diff
@@ -0,0 +1,55 @@
+use std *
+
+export def test_iter_cmd [] {
```

Comment:
Why do you need this? In most cases it is better to export the individual test commands.